### ISSUE TITLE
Save individual features

### ISSUE THREAD

**Issue by Ray-0403** [martineastwood/featuristic#2]
Hi, if after selecting, I only found 6 out of 100 features being really useful, how can I save only these 6 features and use them to compute features on other dataframe (maybe in real time too), maybe there are simply ways to do it by only modifying few lines of source code?

**Comment by martineastwood** [comment]
So would you like to use the `GeneticFeatureSynthesis.transform` function to compute the features on a new dataframe but only for the six features you are interested in?

**Comment by Ray-0403** [comment]
Yeah, that's what I mean, defining a subset of interested features and save it use pickle or other method.

**Comment by Ray-0403** [comment]
Also I checked the source code, fitness function is very easy to be modified since it takes pd series as input, but it's hard to add time series symbolic functions, since the function code takes float as input not pd series... right? 

If I wanna use my custom fitness function, but don't wanna use parsimony coefficient as a penalty, can u apply a limit like max depth to ur library? For example, any program cant have a depth over 20 if I set max depth to 20.
